gms/factories/build.gradle

```diff
@@ -9,8 +9,10 @@ dependencies {
   compile externalDependency.lombok
   compile externalDependency.servletApi
   compile externalDependency.springBeans
+  compile externalDependency.springBootAutoconfigure
```

Comment:
GMS is not a Spring Boot app. Is it safe to take a dependency to Spring Boot from GMS? Did you have a chance to validate these changes on your local?

Reply:
Yes; I verified that GMS starts and connects using `KafkaProperties` without any errors in the log.